bpf/deserialization_to_bpf_map/deserialization_to_bpf_map.c

```diff
@@ -1359,9 +1355,8 @@ int inner_map_create_all(struct bpf_map_info *inner_info)
         g_inner_map_mng.inner_maps[i].map_fd = fd;
     }
 
-    if (i < MAX_OUTTER_MAP_ENTRIES) {
-        printf("inner_map_create %d failed:%d\n", i, fd);
-    }
+    if (i < MAX_OUTTER_MAP_ENTRIES)
+        LOG_WARN("[warning]inner_map_create (%d->%d) failed:%d, errno:%d\n", i, MAX_OUTTER_MAP_ENTRIES, fd, errno);
```

Comment:
donot we need to  return erro nunber? ANd what will happend if later we need to r/w on the map